lib/instruction-parser.ts

```diff
@@ -1,7 +1,14 @@
-export { getPackagesFromRunInstructions, DockerFilePackages };
+export { getPackagesFromRunInstructions, DockerFilePackages, DockerFileLayers };
 
 interface DockerFilePackages {
   [packageName: string]: {
+    id: string,
```

Comment:
Unfortunately this is required to later map packages to layer ids. I would want to replace `instruction` with `id` altogether, but that would be a breaking change that fixing will require a lot of effort that will also slip to other repositories.